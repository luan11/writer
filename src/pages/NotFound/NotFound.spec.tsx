import { render, screen } from '@testing-library/react';

import { NotFound } from '.';

describe('<NotFound />', () => {
  it('should render NotFound correctly', () => {
    render(<NotFound />);

    expect(screen.getByRole('heading', { name: '404 - Página não encontrada' }))
      .toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = render(<NotFound />);

    expect(container.firstChild).toMatchSnapshot();
  });
});