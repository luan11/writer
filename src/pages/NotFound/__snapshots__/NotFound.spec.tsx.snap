// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should match snapshot 1`] = `
<h2
  style="text-align: center;"
>
  404 - Página não encontrada
</h2>
`;
